js/src/forum/SearchSources/DiscussionsSearchSource.tsx

```diff
@@ -54,7 +54,7 @@ export default class DiscussionsSearchSource implements SearchSource {
       const mostRelevantPost = discussion.mostRelevantPost();
 
       return (
-        <li className="DiscussionSearchResult" data-index={`${this.type}${discussion.id()}`} key={`${this.type}${discussion.id()}`}>
+        <li className="DiscussionSearchResult" data-index={`${this.type}${discussion.id()}`}>
```

Comment:
@imorland what's the reason for removing the key?

Reply:
Mithril complains.. We use the `data-index` anyways, not the `key` in this instance